src/Tinter.js

```diff
@@ -376,8 +383,7 @@ class Tinter {
                     const attr = this.svgAttrs[k];
                     for (let l = 0; l < this.keyHex.length; l++) {
                         if (tag.getAttribute(attr) &&
-                            tag.getAttribute(attr).toUpperCase() === this.keyHex[l]) 
-                        {
+                            tag.getAttribute(attr).toUpperCase() === this.keyHex[l]) {
```

Comment:
the style we would generally go with is
```
if (reallyLongstuff &&
    otherReallyLongStuff
) {
    bla

Reply:
says who, though? i've never used that style in my life, and it's not in our code style (https://github.com/matrix-org/matrix-react-sdk/blob/master/code_style.md), and I don't understand why we are enforcing lint rules which are more extreme than our code style!

Reply:
I've never seen that, actually. We really ought to update the linting configuration to reflect that.

Reply:
I don't find it that bad, but then I do have the linting built into my IDE, so there's no extra "appeasing" step in general.